sdk/spring/spring-cloud-azure-autoconfigure/src/main/resources/META-INF/additional-spring-configuration-metadata.json

```diff
@@ -1568,6 +1568,13 @@
       "type": "java.lang.String",
       "description": "Name of the Azure cloud to connect to.",
       "sourceType": "com.azure.spring.cloud.service.implementation.passwordless.AzurePasswordlessProperties"
+    },
+    {
+      "name": "spring.datasource.azure.passwordless-enabled",
+      "type": "java.lang.Boolean",
+      "description": "Whether to enable supporting Azure identity token credentials.",
```

Comment:
```suggestion
      "description": "Whether to enable passwordless connections to Azure databases by using Azure Active Directory token credentials.",
```

Reply:
Good point, I think we need to update our docs, codes as well.

Reply:
> 

Should we add `OAuth2` before the `token credentials`